File: gcd.py
def gcd(a,b):
    if a==0:
        return b
    else :
        while (b!=0):
            r=a%b
            a=b
            b=r
        return a
print("Enter  greatest common divisor\n")
a=int(input('First Number'))
b=int(input("Second Number"))
ans = gcd(a,b)
print(f"The required answer is {ans} ")

File: gcd.c
#include<stdio.h>
int  gcd (int a , int b )
{
int r;
if (b==0) 
{
    return a;
}
else
{
    while(b!=0){
        r=a%b;
        a=b;
        b=r;

    }
    return a;
}

}
int main (){
    int a ,b;
    printf("Enter\n ");
    scanf("%d%d",&a,&b);
    printf("%d",gcd(a,b));
return 0;
}
